docs/DeveloperGuide.md

```diff
@@ -287,32 +287,137 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `FAPro` and the **Actor** is the `Financial Advisor`, unless specified otherwise)
 
+**Use case: Add a person**
+
+**MSS**
+
+1.  Financial Advisor requests to list persons
+2.  FAPro shows a list of persons
+3.  Financial Advisor requests to add a new person
+4.  FAPro adds the person to the address book based on the specified parameter (name, address, phone number, email address, occupation, and tag)
+
+    Use case ends.
+
+**Extensions**
+
+* 3a. The parameter is provided in an invalid format.
+
+    * 3a1. FAPro shows an error message: "Invalid command format!"
+  
+        Use case resumes at step 2.
+
+* 3b. The parameter is specified multiple times.
+
+    * 3b1. FAPro shows an error message: "The parameter can only be specified once!"
+  
+        Use case resumes at step 2.
+
+* 3c. The person's name is the same as the existing name in the address book.
+
+    * 3c1. FAPro shows an error message: "This person already exists in the address book"
+
+        Use case resumes at step 2.
+
+    
 **Use case: Delete a person**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Financial Advisor requests to list persons
+2.  FAPro shows a list of persons
+3.  Financial Advisor requests to delete a specific person in the list
+4.  FAPro deletes the person
 
     Use case ends.
 
 **Extensions**
 
 * 2a. The list is empty.
 
-  Use case ends.
+    Use case ends.
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. FAPro shows an error message:  “Sorry, that value is not accepted! Please specify the index of the person you would like to delete! It should be non-negative and within the address book!”
 
-      Use case resumes at step 2.
+        Use case resumes at step 2.
+
+**Use case: Find persons**
+
+**MSS**
+
+1.  Financial Advisor requests to list persons
+2.  FAPro shows a list of persons
+3.  Financial Advisor requests to find a specific persons in the list by name
```

Comment:
Possible typo? "persons" -> "person"

Reply:
<img width="671" alt="Screenshot 2023-10-03 130920" src="https://github.com/AY2324S1-CS2103T-W09-1/tp/assets/96093411/6d116e3e-cf9f-49b9-8bee-3c7b84d669c5">
I follow the format of the current Developer Guide. So, should I change all persons -> person or just change the number 3?

Reply:
Just number 3. The other uses of persons are fine.

Reply:
Noted. Thank you @bhnuka @limzhenwy